### PR TITLE
added ice effective radius product from Griesche et al. 2020

### PR DESCRIPTION
**Pull request by KarlJohnsonnn**
Griesche et al. 2020 ice effective radius calculation

### CODE REVIEW

**Comment by KarlJohnsonnn** [comment]
Hi, 

I need help, somehow the checks were not successful, see:

`Run mypy --ignore-missing-imports cloudnetpy/ tests/
[6](https://github.com/actris-cloudnet/cloudnetpy/runs/6507336127?check_suite_focus=true#step:6:7)
cloudnetpy/products/ier.py:325: error: Argument "comment" to "MetaData" has incompatible type "Sequence[str]"; expected "Optional[str]"
[7](https://github.com/actris-cloudnet/cloudnetpy/runs/6507336127?check_suite_focus=true#step:6:8)
cloudnetpy/products/ier.py:331: error: Argument "comment" to "MetaData" has incompatible type "Sequence[str]"; expected "Optional[str]"
[8](https://github.com/actris-cloudnet/cloudnetpy/runs/6507336127?check_suite_focus=true#step:6:9)
cloudnetpy/products/ier.py:335: error: Argument "comment" to "MetaData" has incompatible type "Sequence[str]"; expected "Optional[str]"
[9](https://github.com/actris-cloudnet/cloudnetpy/runs/6507336127?check_suite_focus=true#step:6:10)
cloudnetpy/products/ier.py:342: error: Argument "comment" to "MetaData" has incompatible type "Sequence[str]"; expected "Optional[str]"
[10](https://github.com/actris-cloudnet/cloudnetpy/runs/6507336127?check_suite_focus=true#step:6:11)
Found 4 errors in 1 file (checked 100 source files)
[11](https://github.com/actris-cloudnet/cloudnetpy/runs/6507336127?check_suite_focus=true#step:6:12)
Error: Process completed with exit code 1.`


I used the same notation as for iwc.py comments, a dict with key and tuple of comments. Not sure why it fails here but not in `der` or other products.

**Comment by tukiains** [comment]
I believe you need to at least `git rebase master` and to fix merge conflicts first.

**Comment by KarlJohnsonnn** [comment]
I did `git rebase master` and followed along. The merge conflicts were resolved and changes are pushed but still checks are not successful?

File: cloudnetpy/products/ier.py [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+
+COMMENTS = {
+    "ier": (
+        "Ice effective radius Hogen et al. 2006."
+    ),
+    "ier_retrieval_status_comment": (
+        "This variable describes whether a retrieval was performed for each\n"
+        "pixel, and its associated quality, in the form of 8 different classes.\n"
+        "The classes are defined in the definition and long_definition attributes. \n"
+        "The most reliable retrieval is that without any rain or liquid\n",
```
Comment (siiptuo) [inline cloudnetpy/products/ier.py:266]: There's a misplaced comma here and fixing this should fix the type error:
```suggestion
        "The most reliable retrieval is that without any rain or liquid\n"
```

File: cloudnetpy/products/ier.py [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+        "cloud beneath, indicated by the value 1, then the next most reliable \n"
+        "is when liquid water attenuation has been corrected using a microwave\n"
+        "radiometer, indicated by the value 3, while a value 2 indicates that \n"
+        "liquid water cloud was present but microwave radiometer data were not\n"
+        "available so no correction was performed. No attempt is made to retrieve \n"
+        "ice water content when rain is present below the ice; this is\n"
```
Comment (siiptuo) [inline cloudnetpy/products/ier.py:272]: The last sentence seems to be cut off: "this is..."

**Review by siiptuo — CHANGES_REQUESTED** [review]

**Comment by tukiains** [comment]
The errors come from the `cloudnetpy-qc` package. We are fixing these right now.. Let's see if the tests pass soon.

**Comment by KarlJohnsonnn** [comment]
@siiptuo Thanks for spotting the typos! It seems to work now.

**Comment by tukiains** [comment]
Great! We well take a closer look at it now.

**Comment by tukiains** [comment]
Thanks again @KarlJohnsonnn! Good work! :+1:

**Pull request merged**